cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -436,13 +377,12 @@ def test_course_listing_with_actions_in_progress(self):
             )
 
         # verify return values
-        for method in (_accessible_courses_list_from_groups, _accessible_courses_iter):
-            def set_of_course_keys(course_list, key_attribute_name='id'):
-                """Returns a python set of course keys by accessing the key with the given attribute name."""
-                return set(getattr(c, key_attribute_name) for c in course_list)
-
-            found_courses, unsucceeded_course_actions = method(self.request)
-            self.assertSetEqual(set_of_course_keys(courses + courses_in_progress), set_of_course_keys(found_courses))
-            self.assertSetEqual(
-                set_of_course_keys(courses_in_progress), set_of_course_keys(unsucceeded_course_actions, 'course_key')
-            )
+        def set_of_course_keys(course_list, key_attribute_name='id'):
```

Comment:
Tiny nit: could you prepend this method name with `_` to mark it as an internal method?

`_set_of_course_keys()`